tests/python/test.py

```diff
@@ -459,6 +459,20 @@ def test_layer_norm(self):
         self.assertTrue(np.allclose(y.npvalue(), y_np_value))
 
 
+class TestSlicing(unittest.TestCase):
+
+    def test_layer_norm(self):
```

Comment:
The name of this function is wrong, but other than that this is great! I'm happy to merge once this is fixed.